### ISSUE TITLE
Babel 7.2.0 not suported

### ISSUE THREAD

**Issue by TheAifam5** [babel/babel-loader#739]
*Delete what isn't applicable below*
**I'm submitting a bug report**

**Webpack Version:**
4.26.1

**Babel Core Version**:
7.2.0

**Babel Loader Version**:
8.0.4

**Please tell us about your environment:**
ArchLinux

**Current behavior:**
Thows an error while building:
```
Module build failed (from ../node_modules/babel-loader/lib/index.js):
TypeError: path.isClassPrivateMethod is not a function
    at verifyUsedFeatures (node_modules/@babel/helper-create-class-features-plugin/lib/features.js:46:12)
    at PluginPass.Class (node_modules/@babel/helper-create-class-features-plugin/lib/index.js:69:42)
....
```

**Expected/desired behavior:**
Should work.

**Comment by nicolo-ribaudo** [comment]
Can you run `npx nls why @babel/types` to check the installed versions?

**Comment by TheAifam5** [comment]
It's resolved: https://github.com/babel/babel/issues/9120

**Comment by nicolo-ribaudo** [comment]
Fixed by babel/babel#9121, it will be released soon.
